pixi.toml

```
@@ -16,6 +16,10 @@ linux = "4.18"
 # This is a single place where CMAKE_BUILD_TYPE is defined, change its value here and
 # delete .build or .build-ros2 to change build type
 CMAKE_BUILD_TYPE = "Release"
+# ccache support
+CMAKE_C_COMPILER_LAUNCHER="ccache"
+CMAKE_CXX_COMPILER_LAUNCHER="ccache"
+CCACHE_BASEDIR="$PIXI_PROJECT_ROOT"
```

FYI I think this ends up being a no-op on Windows, because you need to use the percent syntax:


```suggestion
CCACHE_BASEDIR="%PIXI_PROJECT_ROOT%"
```